### PR TITLE
all: fix golangci-lint errors

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Fixes "indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)"

### CODE REVIEW

File: pkg/aws/instance_info.go [hunk 1]
```diff
@@ -124,34 +124,34 @@ func (types *InstanceTypes) SyntheticInstanceInfo(instanceTypes []string) (Insta
...
-			result.InstanceStorageDevices = min(result.InstanceStorageDevices, info.InstanceStorageDevices)
-		}
-		result.InstanceType = "<multiple>"
-		return result, nil
+		result.VCPU = min(result.VCPU, info.VCPU)
+		result.Memory = min(result.Memory, info.Memory)
+		result.InstanceStorageDeviceSize = min(result.InstanceStorageDeviceSize, info.InstanceStorageDeviceSize)
+		result.InstanceStorageDevices = min(result.InstanceStorageDevices, info.InstanceStorageDevices)
 	}
+	result.InstanceType = "<multiple>"
```
Comment (szuecs) [inline pkg/aws/instance_info.go:153]: So if there is only one instance type, we have now `<multiple>`

Comment (szuecs) [reply pkg/aws/instance_info.go:153]: ah no in line 125 we check that we have one and return early. This means you can move result.InstanceType to the top.

Comment (AlexanderYastrebov) [reply pkg/aws/instance_info.go:153]: There is a check `if len(instanceTypes) == 1` above. The change is equivalent as it just removes else branch where other if branches have return

Comment (AlexanderYastrebov) [reply pkg/aws/instance_info.go]: Fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**